.pre-commit-hooks.yaml

```
@@ -1,6 +1,16 @@
--   id: check
-    name: FawltyDeps
-    description: Python dependency checker
-    entry: fawltydeps
+-   id: check-undeclared
+    name: FawltyDeps-undeclared
+    description: "FawltyDeps: Check for undeclared dependencies"
+    entry: fawltydeps --check-undeclared
     language: python
+    files: ".*\\.py|ipynb"
+    args:
+      - --code
+    pass_filenames: true
+-   id: check-unused
+    name: FawltyDeps-unused
+    description: "FawltyDeps: Check for unused dependencies"
+    entry: fawltydeps --check-unused
+    language: python
+    files: ".*\\.py|ipynb"
     pass_filenames: false
```

Here, I think we want to do the converse of the above: Pass `--deps` plus the dependency declaration files that have staged changed. Something like this:

```suggestion
    args: [--deps]
    files: "(.*requirements.*\\.(txt|in)|pyproject\\.toml|setup\\.(py|cfg))"
```